src/editor_serie/EditorSerieController.cpp

```diff
@@ -32,6 +32,7 @@ const std::string EditorSerieController::EDITOR_SERIE_BUTTON_OPEN_TEXT = "Open";
 const std::string EditorSerieController::EDITOR_SERIE_BUTTON_SAVE_TEXT = "Save";
 const std::string EditorSerieController::EDITOR_SERIE_BUTTON_ADD_TEXT = "Add";
 const std::string EditorSerieController::EDITOR_SERIE_BUTTON_EXIT_TEXT = "Exit";
+const char EditorSerieController::SERIES_DIRECTORY[] = "data/series";
```

Comment:
use const char* instead of const char[], the pointed array won't change, and const char* is faster